### PR TITLE
override toString on bumps to make them human readable

### PR DESCRIPTION
**Pull request by pdalpra**

### CODE REVIEW

File: src/main/scala/Version.scala [hunk 1]
```diff
@@ -8,11 +8,11 @@ object Version {
   }
 
   object Bump {
-    object Major extends Bump { def bump = _.bumpMajor }
-    object Minor extends Bump { def bump = _.bumpMinor }
-    object Bugfix extends Bump { def bump = _.bumpBugfix }
-    object Nano extends Bump { def bump = _.bumpNano }
-    object Next extends Bump { def bump = _.bump }
+    object Major extends Bump { def bump = _.bumpMajor; override def toString = "Major" }
```
Comment (jozic) [inline src/main/scala/Version.scala]: how about just making them case objects? 
like `case object Major extends Bump { def bump = _.bumpMajor }`

Comment (pdalpra) [inline src/main/scala/Version.scala]: Good point !

**Comment by pdalpra** [comment]
@jozic Fixed ! Since it somewhat linked to #189, making them case objects make it easier to pattern-match on them, as what I am doing requires to impl an Ordering for `Bump`s :)

**Comment by pdalpra** [comment]
friendy bump ? :)

**Pull request merged**